sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -664,6 +666,8 @@ def get_active_apsel_hostlane(self):
         '''
         This function returns the application select code that each host lane has
         '''
+        if (self.is_flat_memory()):
+            return 'N/A'
```

Comment:
Example from a non-flat memory result would be 
{'ActiveAppSelLane1': 1, 'ActiveAppSelLane2': 1, 'ActiveAppSelLane3': 1, 'ActiveAppSelLane4': 1, 'ActiveAppSelLane5': 1, 'ActiveAppSelLane6': 1, 'ActiveAppSelLane7': 1, 'ActiveAppSelLane8': 1}.

Could you keep all the keys in the dictionary and make values 'N/A'?